.travis.yml

```diff
@@ -59,3 +59,8 @@ script:
   echo;
   echo "Running PHP-CS-Fixer";
   composer ci:php:fixer;
+
+- >
+  echo;
+  echo "Running Psalm";
+  composer ci:php:psalm;
```

Comment:
Generally I'd say yes, but I've found that php-cs-fixer will break indented array shape docblock values.

Would defer this until stage two of psalm checking (all issues resolved) is complete or #742 is resolved (since array shapes are used during static analysis w/ psalm)

Reply:
We're running php-cs-fixer in dry-run mode, i.e., it won't change any code.

Reply:
> We're running php-cs-fixer in dry-run mode, i.e., it won't change any code.

do you manually fix issues identified by php-cs-fixer?

Reply:
> do you manually fix issues identified by php-cs-fixer?

Yes (in the sense that we'll need to commit those changes so the build will be green, and that there's no bot amending the failing PRs).

Reply:
Travis-CI rejects commits that cause issues.  I know that @oliverklee has used the fixer automatically, both before we had this check, and as we introduced it, but locally.  (I could find a reference, but I've just used up all my refrence-finding time for this month.)

Having a local `composer ci` script to do it could be useful, but mostly only if we're introducing new checks (or upgrading to a new version that does).  On the backburner maybe.

(Personally, I'd rather make the suggested edits manually, if it's just a new feature I'm indroducing and I've fallen foul of some rules, so I can be clear where I'm at.)

Reply:
> > do you manually fix issues identified by php-cs-fixer?
> 
> Yes (in the sense that we'll need to commit those changes so the build will be green, and that there's no bot amending the failing PRs).

my point being that if `php-cs-fixer --fix` makes larges swathes of changes, indented psalm array shapes being removed can be inadvertently overlooked & committed (the workaround is to un-indent/nest the array shape from multi-line to a single line).

Reply:
I'm still not sure I get your point. What's the exact problem you see, or what do you suggest?

Reply:
I *occasionally* run psalm after php-cs-fixer on the basis that php-cs-fixer may identify an issue, so I take out the `--dry-run`, which may then cause an issue in psalm.

tl:dr; either order is fine, caveat coder.